elements/pl-workspace/pl-workspace.py

```diff
@@ -1,6 +1,12 @@
 import chevron
 
 
+def add_format_error(data, error_string):
+    if '_files' not in data['format_errors']:
+        data['format_errors']['_files'] = []
+    data['format_errors']['_files'].append(error_string)
+
+
 def render(element_html, data):
```

Comment:
Add a check to only render in the question panel, like this: https://github.com/PrairieLearn/PrairieLearn/blob/d4306520a7c9cff858aea21b4298200baeec00dd/elements/pl-file-editor/pl-file-editor.py#L47